### DISCUSSION TITLE
Playlist maps' star rating doesn't account for mods

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#21345] [Q&A]
When I make playlists that have maps where you are required to play them with HR or with DT, the star rating of those maps is unaffected and doesn't increase like it is supposed to. 

For example, here's a map that has DT enabled. This map appears to be 4.8*, but it's actually 4.8* without DT. With DT, this map is 6.99*, yet that is not what is shown as the actual star rating.

![image](https://user-images.githubusercontent.com/74088315/202959362-d24d372d-6c4b-49e6-9e7f-a6bde3614c49.png)

This also causes the star rating range of the playlist to be inaccurate. I made a playlist of the OWC 2022 Grand Finals mappool, and the lowest star rating map when mods are taken into consideration is 6.37*, but the playlist says that it's 4.8*, which is completely wrong.

![image](https://user-images.githubusercontent.com/74088315/202959796-3b8eb8fc-ec18-4af5-80ad-4c037925dfab.png)

The star rating makes sense when the maps are to be played with no mods or have optional freemods, but if you have to play a map with a given mod (such as HR or DT), the star rating should match what it's supposed to be with that mod enabled.

**Comment by smoogipoo** [comment]
This is going to be a server-side limitation. I'm not yet sure how the fix for this is going to look like.
